vspace/src/main/java/edu/asu/diging/vspace/core/factory/ISlideFactory.java

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
please don't commit changes to files that are not really changes (and also there shouldn't be an empty line here).